satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -377,6 +310,75 @@ read all (
     orderby asc api_key.name
 )
 
+//-----stripe_db----//
+
+// add payment method id to indentify if it has been set up
+model user_payment (
+    key    user_id
+    unique customer_id
+
+    field user_id       user.id   cascade
+    field customer_id   blob
+
+    field created_at    timestamp ( autoinsert )
+)
+
+create user_payment ( )
+
+read one (
+    select user_payment
+    where  user_payment.user_id = ?
+)
+
+// add is default, make composite key
+model project_payment (
+    key project_id
+
+    field project_id        project.id              cascade
+    field payer_id          user_payment.user_id    cascade
+    field payment_method_id blob
+
+    field created_at        timestamp  ( autoinsert )
+)
+
+create project_payment ( )
+
+read one (
+    select project_payment
+    where project_payment.project_id = ?
+)
+read one (
+    select project_payment
+    where project_payment.payer_id = ?
+)
+
+model project_invoice_stamp (
+    key    project_id start_date end_date
+    unique invoice_id
+
+    field  project_id project.id  cascade
+    field  invoice_id blob
+
+    field  start_date timestamp
+    field  end_date   timestamp
+
+    field  created_at timestamp
+)
+
+create project_invoice_stamp ( )
+
+read one (
+    select project_invoice_stamp
+    where  project_invoice_stamp.project_id = ?
+    where  project_invoice_stamp.start_date = ?
+)
+// pagination ?
+read all (
+    select project_invoice_stamp
+    where  project_invoice_stamp.project_id = ?
+    orderby desc project_invoice_stamp.start_date
+)
+
```

Comment:
no changes in definitions? no need for migration?

Reply:
Yup, but I am not sure, will check